src/pages/Order/components/Recipient.tsx

```diff
@@ -0,0 +1,80 @@
+import Divider from "@/components/common/Divider";
+import styled from "@emotion/styled";
+import type { ChangeEvent } from "react";
+import Input from "@/pages/Order/components/Input";
+
+interface RecipientProps {
+  name: string;
+  onChangeName: (event: ChangeEvent<HTMLInputElement>) => void;
+  errorMsgName: string | null;
+  phone: string;
+  onChangePhone: (event: ChangeEvent<HTMLInputElement>) => void;
+  errorMsgPhone: string | null;
+  quantity: number;
+  onChangeQuantity: (event: ChangeEvent<HTMLInputElement>) => void;
+  errorMsgQuantity: string | null;
```

Comment:
```suggestion
  name: ComponentProps<typeof Input>,
  phone: ComponentProps<typeof Input>,
  quantity: ComponentProps<typeof Input>,
```
name, phone, quantity 에 대한걸 namespace로 묶어서 받아오면 어떨까요?

Reply:
개인적으로 컴포넌트의 렌더링이 상위에서 하위로 전파되기 때문에
이 컴포넌트 내부에 state를 선언하고, 바깥에는 onSubmit 같은 이벤트로 전송하는 시점에 필요한 정보만 알려주면 어떨까 싶어요 ㅎㅎ